Packs/Intezer/Playbooks/playbook-Intezer_-_Analyze_File_and_URL.yml

```diff
@@ -1,6 +1,6 @@
 id: Intezer - Analyze File and URL
 version: -1
-fromversion: 6.5.0
+fromversion: 6.2.0
```

Comment:
@efelmandar - Do we support this?

Reply:
Probably my mistake; I changed it back to 6.5.0